packages/iov-cosmos/src/cosmoscodec.ts

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
why is this required?

Reply:
See `bytesToSign`: amino needs camelcase.